metadata-ingestion/src/datahub/ingestion/source/iceberg/iceberg.py

```diff
@@ -248,10 +243,31 @@ def _process_dataset(dataset_path: Identifier) -> Iterable[MetadataWorkUnit]:
                 LOGGER.warning(
                     f"Iceberg Rest Catalog server error (500 status) encountered when processing table {dataset_path}, skipping it."
                 )
+            except ValueError as e:
+                if "Could not initialize FileIO" not in str(e):
+                    raise
+                self.report.warning(
```

Comment:
may you have forgot the redundant `LOGGER.warning` for this except case?

Reply:
`warning` method will automatically log, unlike `report_warning`. I should change all lines to use `warning` I think, see: https://github.com/datahub-project/datahub/blob/5309ae00141875cca6be64e587a94efa854f448e/metadata-ingestion/src/datahub/ingestion/api/source.py#L247-L267